charts/kubeai/Chart.yaml

```diff
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.7.0
+version: 0.7.1
```

Comment:
note, you can leave this in this PR, but in the future, no need to bump the chart versions. I handle this as part of releases.